src/OgRoleInterface.php

```diff
@@ -51,4 +51,42 @@
    */
   public function setId($id);
 
+  /**
+   * Returns default properties for the given default OG role name.
+   *
+   * These are the properties used to create the two roles that are required by
+   * every group: the 'member' and 'non-member' roles.
```

Comment:
`member` and `non-member` are the ones provided by OG core, but extending modules may decide to add other default roles. 

My confusion is probably due to the fact we have `GroupManager::getDefaultRoles`.

So, maybe it's worth changing this some like `getRequiredDefaultRole` (for the member and non-member) vs `getDefaultRoles` for getting all the OG required roles + the ones defined by implementing modules?